### ISSUE TITLE
Why is my result on the Dota validation set only 0.28

### ISSUE THREAD

**Issue by Arui03** [VisionXLab/Point2RBox-v3#19]
Why is my result on the Dota validation set only 0.28, Is this normal? I have over 0.6 on the test set. Could you please teach me how to adjust the parameters? Thank you very much.

**Comment by johnson-magic** [comment]
Could you provide me with your training log and the content of the test result feedback email ? @Arui03

**Comment by Arui03** [comment]
Thank you for your reply. However, due to some reasons, my computer is unable to open the file to view more information. These are the details I can provide at present. Once again, thank you for your response.
<img width="1629" height="360" alt="Image" src="https://github.com/user-attachments/assets/2ea6c3cd-37aa-4cb9-8cc8-087af91c39e5" />

[Task1.zip](https://github.com/user-attachments/files/26456510/Task1.zip)

**Comment by johnson-magic** [comment]
Retraining should be fairly quick, probably just 10+ hours. Please provide me with the training log and the .pt file from the last epoch — I think that should help us pinpoint the cause. @Arui03

**Comment by Arui03** [comment]
The document format is not supported, so I'm using the cloud storage to send it to you. https://pan.baidu.com/s/1wRgS8QFzIWDluK9T9GyxJw 提取码: 247u

**Comment by johnson-magic** [comment]
Could you please provide all the training logs and config files located in ./work_dirs/point2rbox_v3-1x-dotav1-0 ? @Arui03

**Comment by johnson-magic** [comment]
I tried offline evaluating the model you provided, and its metrics on trainval are as follows:
<img width="499" height="399" alt="Image" src="https://github.com/user-attachments/assets/c3c8bb29-b37c-4245-bb8a-ff0c66956482" />
The official test feedback email is as follows: 

<img width="1261" height="126" alt="Image" src="https://github.com/user-attachments/assets/a3a6a1d7-a291-4fe9-b052-4714a9e22abe" />

I'm not sure if your training data is processed correctly. Here is my processed training data. You can try downloading and using it. See if training with this dataset yields normal metrics.

https://pan.quark.cn/s/56c4793f7c9d

@Arui03

**Comment by mingqian-233** [comment]
The document format is not supported, so I'm using cloud storage to send it to you. https://pan.baidu.com/s/1wRgS8QFzIWDluK9T9GyxJw Extraction code: 247u Since this document format is not supported, I’m using cloud storage to send it to you. https://pan.baidu.com/s/1wRgS8QFzIWDluK9T9GyxJw Extraction code: 247u

I think it is very likely that this result was caused by your not processing the data according to **```data.md```** in the root directory. If there are any new developments, please let us know. Thank you!

**Comment by Arui03** [comment]
thank you for your reply and help.  I will make the changes and checks as per your suggestions

**Comment by johnson-magic** [comment]
It has been a week since we last heard from you, and since we haven't received any new feedback, we will temporarily close this issue. If our work has been helpful to you, please feel free to follow us and give our repo a star. @Arui03
